app/controllers/admin/campaigns_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class Admin::CampaignsController < Admin::ApplicationController
+  expose(:campaigns) { Campaign.order(created_at: :desc).page params[:page] }
+  expose(:campaign) do
+    params[:id] ? Campaign.find(params[:id]) : Campaign.new(campaign_params)
+  end
+  expose(:events) { RsvpEvent.order(happens_at: :desc) }
+
+  before_action :deny_if_sent, only: %i[update destroy]
+
+  def show
+    membership = current_user.memberships.current.first
```

Comment:
flagging this for future refactor - we could have a helper method on `User` that returns the most recent active `Membership` or `raise`, or a Memberships service?

Reply:
Yeah, I considered pulling this out into a `current_membership` method, but decided just to focus on what was needed here for the moment.